Yubico.YubiKey/src/Yubico/YubiKey/Piv/PivSession.cs

```diff
@@ -117,10 +120,12 @@ namespace Yubico.YubiKey.Piv
     ///             management key (hex): 01 02 03 04 05 06 07 08
     ///                                   01 02 03 04 05 06 07 08
     ///                                   01 02 03 04 05 06 07 08
-    ///             
+    ///         </code>
```

Comment:
Had to add these to fix a weird problem where the rest of the remarks section (lines 127-145) was formatted as a code box instead of a regular paragraph.